File: playwrightSpecs/pages/components/homePage/Temperature.js
export class Temperature {
    constructor(page){
        this.page = page;
    }
    get temperatureSection(){
        return this.page.locator('ngx-temperature [tabtitle="Temperature"]')
    }
    get temperatureGauge(){
        return this.page.locator('[tabtitle="Temperature"] ngx-temperature-dragger circle')
    }
}

File: playwrightPy/pages/components/homePage/Temperature.py
from playwright.sync_api import Page, Locator # type: ignore

class Temperature:
    def __init__(self, page:Page):
        self.page=page

    @property
    def temperatureSection(self)-> Locator:
        return self.page.locator('ngx-temperature [tabtitle="Temperature"]')
    
    @property
    def temperatureGauge(self)-> Locator:
        return self.page.locator('[tabtitle="Temperature"] ngx-temperature-dragger circle')